crates/prql-compiler/tests/integration/main.rs

```diff
@@ -348,7 +345,10 @@ fn test_rdbms() {
             }
             let dialect = con.dialect;
             let options = Options::default().with_target(Sql(Some(dialect)));
-            let mut rows = prql_compiler::compile(&prql, &options)
+            let replaced_prql = regex::Regex::new("data_file_root")
+                .unwrap()
+                .replace_all(&prql, &con.data_file_root);
```

Comment:
It would be quite wasteful to do this for every query, but perhaps not a problem? (I was also thinking of doing this only if the filename contains `read_` as a workaround)

Reply:
I think it's OK, and less code than checking for the filename (which not be much cheaper...)

One small change would be to move the Regex creation outside the loop — plausibly that's an order of magnitude more expensive than the replace itself.